package.json

```diff
@@ -2,80 +2,29 @@
   "name": "wa-react-2021-q4-capstone-project",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
```

Comment:
i don't think is a good idea install this huge library just to use some few icons, i prefer if you use it as svg react component or as image whit css.

Reply:
Yes you're right, I'll change it